app/templates/content/framework-digital-services.html

```diff
@@ -0,0 +1,160 @@
+      </p>
+      <p>
+        The basic terms of use are between the government and suppliers following a formal <a href="http://www.ojeu.eu">Official Journal of the European Union (OJEU)</a> procurement process. Suppliers on the Digital Services framework must:
+      </p>
+      <ul>
+        <li>
+          work in <a href="https://www.gov.uk/service-manual/agile/index.html">an agile way</a>
+        </li>
+        <li>
+          work according to guidance in the <a href="https://www.gov.uk/service-manual">Government Service Design Manual</a>
+        </li>
+        <li>
+          provide services that are <a href="https://www.gov.uk/service-manual/start">digital by default</a>
+        </li>
+      </ul>
+      <p>
+        There are over 170 suppliers on the Digital Services framework, who provide all or some of the following roles:
+      </p>
+      <table class="dss-table">
+        <thead><tr>
+        <th>Capabilities</th>
+              <th>Linked roles</th>
+            </tr></thead>
+        <tbody>
+        <tr>
+        <td>Software engineering and ongoing support</td>
+              <td>Developer, technical architect</td>
+            </tr>
+        <tr>
+        <td>Product development and service design</td>
+              <td>Product manager, business analyst</td>
+            </tr>
+        <tr>
+        <td>Agile delivery management</td>
+              <td>Delivery manager, product manager, business analyst</td>
+            </tr>
+        <tr>
+        <td>Front-end design and interaction design</td>
+              <td>Designer</td>
+            </tr>
+        <tr>
+        <td>Content design and development</td>
+              <td>Content designer</td>
+            </tr>
+        <tr>
+        <td>System administration and web operations</td>
+              <td>Web operations, security specialist, database specialist, network specialist</td>
+            </tr>
+        <tr>
+        <td>User research</td>
+              <td>Researcher</td>
+            </tr>
+        <tr>
+        <td>Embedding Agile</td>
+              <td>Agile coach</td>
+            </tr>
+        </tbody>
+      </table>
+      <p>
+        These roles and capabilities may change from iteration to iteration of the Digital Services framework.
+      </p>
+
+
+
+
+      <h2 id="submit-services-to-the-digital-services-framework">
+        <span class="number">4. </span>Submit services to the Digital Services framework
+      </h2>
+      <p>
+        You can apply to submit your services to the Digital Services framework when an <a href="http://www.google.com/url?q=http%3A%2F%2Fwww.ojeu.eu&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNFCtRdMjOQV9-ViCXXermvKHlWr6A">Official Journal of the European Union (OJEU)</a> is open. On average, an OJEU will be open for 6 weeks, every 6 to 9 months. It will be published on <a href="http://ted.europa.eu">Tenders Electronic Daily</a>.
```

Comment:
This Google link looks wrong?  And should both the OJEU and TED links here be external?